spk/rutorrent/Makefile

```diff
@@ -1,15 +1,31 @@
 SPK_NAME = rutorrent
 SPK_VERS = 3.10
-SPK_REV = 12
+SPK_REV = 13
 SPK_ICON = src/rutorrent.png
 DSM_UI_DIR = app
 
-SPK_DEPENDS = "python310:PHP5.6"
+SPK_DEPENDS = "python310:PHP7.3"
```

Comment:
Could totally migrate to PHP7.4 TBH. PHP7.3 only presents the advantage of being the same version as the default one shipped at the level of DSM 7 so it's homogeneous.

Reply:
Migrated to 7.4.